src/main/resources/application.yml

```
@@ -106,3 +107,10 @@ banner:
     default-minutes: 2880 # ←신청 슬롯 기본 락 유지시간 (분)  // BannerApplicationService @Value
     cleanup:
       delay: 300000 #  만료락 해제 주기(밀리초)
+
+
+# 로깅 설정
+logging:
+  level:
+    com.fairing.fairplay.ai.client.GoogleGeminiClient: DEBUG
+    com.fairing.fairplay.ai.rag.service: DEBUG
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)